mobileversion.html

```diff
@@ -0,0 +1,248 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+   <link rel="stylesheet" href="mobileversion.css">
+   <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.googleapis.com">
+   <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+   <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+   <title>Mobile Version Website</title>
+</head>
+
+<body>
+
+   <header class="header">
+      <nav class="nav-bar">
+         <a class="log" href="#">BW</a>
+         
+         <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+         </div>
+      </nav>
+   </header>
+   <section>
+      <div class="flex-container">
+         <div class="item5">
+         </div>
+         <div class="item6">
+            <div>
+               <h1 class="mariane">I'm Bryan<br>
+                  Glad to see<br>
+                  you!<br>
+               </h1>
+            </div>
+            <div>
+               <p class="text">
+                  I’m a software developer! I can help you<br>
+                  build a product , feature or website Look <br>
+                  through some of my work and<br>
+                  experience! If you like what you see and <br>
+                  have a project you need coded, don’t<br>
+                  hestiate to contact me.
+               </p>
+            </div>
+            <div>
+               <h2 class="cap">LET’S CONNECT</h2>
+               <ul class="lass">
+                  <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                  <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+                  <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+                  <li><a href="#"><i class='fab fa-github'></i></a></li>
+                  <li><a href="#"><i class='fab fa-angellist'></i></a></li>
+               </ul>
+            </div>
+         </div>
+      </div>
+   </section>
+   <section>
+      <div id="Portfolio" class="containe">
+         <div class="item1">
+            <img src="https://cdn.dribbble.com/users/472534/screenshots/6591791/blog_drbl.png" alt="snapshoot">
+            <div class="ton">
+               <h2 class="tonic">Tonic</h2>
+            </div>
+            <div class="sec">
+               <ul class="mainp">
+                  <li class="canopy">CANOPY</li>
+                  <li class="mid">&#8226;</li>
+                  <li class="back">Back End Dev</li>
+                  <li class="mid">&#8226;</li>
+                  <li class="year">2015</li>
+                  </ul>

```

Comment:
- [ ] [Optional] You should stick to the design as much as possible. I would suggest you please have more spacing in the middle of the li tag. See the image below for reference
### Your design
![your](https://user-images.githubusercontent.com/81727988/205064409-6085d884-f0b5-44cb-8c20-174d1b5ffb50.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/205064435-2a388bda-8f0d-45dc-b19b-7a8fc4b0fedc.png)